app/src/main/java/com/sun/homecinema/utils/StringExt.kt

```diff
@@ -0,0 +1,9 @@
+package com.sun.homecinema.utils
+
+import java.text.SimpleDateFormat
+import java.util.*
+import java.util.Locale.*
+
+fun String.isAfterNow(): Boolean =
+    SimpleDateFormat("yyyy-MM-dd", getDefault())
+        .parse(this)?.time ?: 0 > Date().time
```

Comment:
có một câu hỏi là khi nào k nên dùng extension function ?

Reply:
Em nghĩ là khi phương thức đó không phải là phương thức chung của cả lớp. như extension kia của em thì có những string nó ko phải là date. nhưng mà em chưa tìm được cách nào phù hợp vì cái Date em lưu là string, nên em xử lý như trên nếu không phải Date luôn trả về false

Reply:
Chúng ta không sử dụng các extension function cho các phương thức không liên quan đến class mà chúng ta cần mở rộng
VD: `fun String.toUserProperties() : UserProperties {
    return UserProperties(this.toUppercase())
}`
 thì rõ ràng đây là chức năng không liên quan đến lớp String và không nên được cung cấp dưới dạng extension vì nó đặc trưng cho business logic 
Mình hỏi vui v thôi =))